### PR TITLE
Update support info

### PR DESCRIPTION
**Pull request by fsundermeyer**
### PR creator: Description

Update support info for SLES and non-SLES hosts.

### PR creator: Are there any relevant issues/feature requests?

* bsc#...
* jsc#SLE-...

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [ ] SLE 15 SP4/openSUSE Leap 15.4 *(current `main`, no backport necessary)*
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
  - [ ] SLE 15 GA
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4
  - [ ] SLE 12 SP3

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

File: xml/containers-support.xml [hunk 1]
```diff
@@ -12,47 +12,63 @@
   <title>Compatibility and support plans</title>
  </info>
  <sect1 xml:id="sec-sle-container-support-matrix">
-  <title>&sle; container support matrix</title>
+  <title>Support information</title>
```
Comment (dariavladykina) [inline xml/containers-support.xml:15]:
```suggestion
  <title>&sle; container support matrix</title>
```

File: xml/containers-support.xml [hunk 2]
```diff
@@ -12,47 +12,63 @@
...
+     <imageobject role="fo">
+      <imagedata fileref="container_support_matrix.png" width="100%" format="PNG"/>
+     </imageobject>
+     <imageobject role="html">
+      <imagedata fileref="container_support_matrix.png" width="100%" format="PNG"/>
+     </imageobject>
+    </mediaobject>
+   </informalfigure>
+   <para>
+    In most scenarios, all SLE containers are expected to be interoperable if
```
Comment (dariavladykina) [inline xml/containers-support.xml:35]:
```suggestion
    In most scenarios, all &slea; containers are expected to be interoperable if
```

**Review by dariavladykina — CHANGES_REQUESTED** [review]
Please see some suggestions for changes

**Review by fsundermeyer — COMMENTED** [review]
Accepted your changes, please merge

**Review by dariavladykina — APPROVED** [review]

**Pull request merged**